0.data_analysis_and_processing/mic_analysis/nbconverted/well_sirna_mic_analysis.py

```diff
@@ -251,12 +247,12 @@
     plt.xticks(fontsize=12)
     plt.yticks(fontsize=12)
 
-    plt.title(f"Distributions of Maximal Information Coeficient (MIC) Scores\nBetween DAPI and {stain} Features per Well", fontsize=16)
+    plt.title(f"Distributions of Maximal Information Coeficient (MIC) Scores\nBetween {comparison_stains[0]} and {comparison_stains[1]} Features per Well", fontsize=16)
 
     plt.xlim(0, 1)
     plt.ylim(0.0, 90.0)
 
-    plt.savefig(distribution_figures_path / f"mic_distributions_dapi_{stain.lower()}_wells.png")
+    plt.savefig(distribution_figures_path / f"mic_distributions_{comparison_stains[0].lower()}_{comparison_stains[1].lower()}_wells.png")
```

Comment:
Nit pick here, consider both resolving the path and making a pathlib path.

Reply:
I don't see a benefit to doing this. I might just keep it as is